drivers/bus/pcix/init.c

```diff
@@ -869,8 +876,6 @@ DriverEntry(IN PDRIVER_OBJECT DriverObject,
     /* Close all opened keys, return driver status to PnP Manager */
     if (KeyHandle) ZwClose(KeyHandle);
     if (ControlSetKey) ZwClose(ControlSetKey);
-    if (ParametersKey) ZwClose(ParametersKey);
-    if (DebugKey) ZwClose(DebugKey);
```

Comment:
And what's the point of moving two `ZwClose` calls whilst keeping others? What is the outcome benefit of doing this?